nat.tf

```diff
@@ -1,10 +1,11 @@
 module "nat_label" {
-  source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.3"
-  namespace = "${var.namespace}"
-  stage     = "${var.stage}"
-  name      = "${var.name}"
-  delimiter = "${var.delimiter}"
-  tags      = "${var.tags}"
+  source     = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.3"
+  namespace  = "${var.namespace}"
+  stage      = "${var.stage}"
+  name       = "${var.name}"
+  delimiter  = "${var.delimiter}"
+  attributes = "${concat(var.attributes,list("nat"))}"
```

Comment:
because the label is used only for tags, this will only be included in the ID tag of `"aws_nat_gateway" "default"`
https://github.com/cloudposse/terraform-aws-dynamic-subnets/blob/e45be0840f9ed76c8d0adf3e325cb0c46141128d/nat.tf#L25

```
tags          = "${module.nat_label.tags}"
```

is this the intended behavior?

Reply:
Yes, this is just to make the tags consistent.